src/components/Menu.vue

```diff
@@ -117,4 +117,16 @@ export default defineComponent({
     };
   },
 });
-</script>
\ No newline at end of file
+</script>
+
+<style scoped>
+ion-menu.md ion-item.selected ion-icon {
+  color: var(--ion-color-secondary);
+}
+ion-menu.ios ion-item.selected ion-icon {
+  color: var(--ion-color-secondary);
+}
```

Comment:
do we need these other selectors if we have the generic ion-item.selected selector?

Reply:
Improved

Reply:
Improved by removing unnecessary selectors.